src/der.rs

```diff
@@ -425,6 +469,75 @@ mod tests {
     use super::DerTypeId;
     use std::prelude::v1::*;
 
+    /// Prepend stuff to `bytes` to put it in a DER SEQUENCE.
+    pub(crate) fn wrap_in_sequence(bytes: &[u8]) -> Vec<u8> {
+        super::asn1_wrap(super::Tag::Sequence, bytes)
+    }
+
+    #[test]
+    fn test_empty() {
+        assert_eq!(vec![0x30, 0x00], wrap_in_sequence(&[]));
+    }
```

Comment:
Since `asn1_wrap` is only avaible with `alloc` these new tests and the `wrap_in_sequence` helper will also need to be alloc-gated.

I think doing that will fix the [--no-default-features CI failure](https://github.com/rustls/webpki/actions/runs/8995896347/job/24733875956?pr=253)

Reply:
Gotcha. Added this in [09a8333](https://github.com/rustls/webpki/pull/253/commits/09a83339c5522c99f6ba7dbb4f30149ebae418f8)!